BBBBroadcast/SampleHandler.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  SampleHandler.swift
+//  BBBBroadcast
+//
+//  Created by Milan Bojic on 6.12.21..
+//
+
+import ReplayKit
+import VideoToolbox
+import WebRTC
+
+class SampleHandler: RPBroadcastSampleHandler {
+    
+    private var signalingClient: SignalingClient?
+    private var webRTCClient = WebRTCClient(iceServers: BBBURL.bbbTurnServers)
+    private let defaults = UserDefaults.init(suiteName: Constants.appGroup)
+    
+    private var javascriptPayload: JavascriptData.Payload?
+    private var isConnected: Bool = false
+    
+    override func broadcastStarted(withSetupInfo setupInfo: [String : NSObject]?) {
+        print("Broadcast has started")
+        // Set webRTC delegate
+        webRTCClient.delegate = self
+        // Parse data from web browser and start websocket connection
+        guard let javascriptData = defaults?.object(forKey: Constants.javascriptData) as? Data else { return }
+        do {
+            let jsData = try JSONDecoder().decode(JavascriptData.self, from: javascriptData)
+            javascriptPayload = jsData.payload
+            setupWebSocketConnection(with: jsData.websocketUrl)
+        } catch (let error) {
+            print("⚡️☠️ Failed to load payload data: \(error.localizedDescription)")
+        }
+    }
+    
+    override func broadcastPaused() {
+        // User has requested to pause the broadcast. Samples will stop being delivered.
+        print("Broadcast has paused.")
+    }
+    
+    override func broadcastResumed() {
+        // User has requested to resume the broadcast. Samples delivery will resume.
+        print("Broadcast has resumed.")
+    }
+    
+    override func broadcastFinished() {
+        // User has requested to finish the broadcast.
+        print("Broadcast has finished.")
+    }
+    
+    override func processSampleBuffer(_ sampleBuffer: CMSampleBuffer, with sampleBufferType: RPSampleBufferType) {
+        print("Broadcast sample buffer received")
+        switch sampleBufferType {
+        case .video:
+            // Handle video sample buffer
+            guard isConnected, let imageBuffer: CVImageBuffer = CMSampleBufferGetImageBuffer(sampleBuffer) else { break }
+//            let pixelFormat = CVPixelBufferGetPixelFormatType(imageBuffer) // kCVPixelFormatType_420YpCbCr8BiPlanarFullRange
+            let timeStampNs: Int64 = Int64(CMTimeGetSeconds(CMSampleBufferGetPresentationTimeStamp(sampleBuffer)) * 1000000000)
+            let rtcPixlBuffer = RTCCVPixelBuffer(pixelBuffer: imageBuffer)
+            let rtcVideoFrame = RTCVideoFrame(buffer: rtcPixlBuffer, rotation: ._0, timeStampNs: timeStampNs)
```

Comment:
Why the `_`?

Reply:
To be honest I saw this line of code in some example and left it as it is. Will check out all the options, and see why it was written like that.